src/main/java/org/broadinstitute/consent/http/models/AuthUser.java

```diff
@@ -15,13 +18,24 @@ public class AuthUser implements Principal {
   public AuthUser() {
   }
 
-  public AuthUser(String email) {
+  public AuthUser(String authToken, String email, String name, String aud) {
+    this.authToken = authToken;
     this.email = email;
+    this.name = name;
+    this.aud = aud;
   }
 
-  public AuthUser deepCopy() {
-    Gson gson = new Gson();
-    return gson.fromJson(gson.toJson(this), AuthUser.class);
+  public AuthUser(String authToken, String email, String name, String aud,
+      UserStatusInfo userStatusInfo) {
+    this.authToken = authToken;
+    this.email = email;
+    this.name = name;
```

Comment:
Same consideration might apply here about getting rid of the `name` here.

Reply:
Name is required to implement `Principal`